### ISSUE TITLE
Create a BetterTabActivity

### ISSUE THREAD

**Issue by erizzo** [mttkay/droid-fu#29]
One base Android Activity that isn't included in the DroidFu extensions is TabActivity. After a quick glance, it looks like the code from BetterDefaultActivity could probably be copied into a new class like:
BetterTabActivity extends TabActivity implements BetterActivity {
...
}

I think it would be a 2-minute addition, not including testing of course.
I was going to do this in my own package for an app I'm currently working on, but BetterActivityHelper.handleApplicationClosing() is not public so I can't call it.

**Comment by mttkay** [comment]
yes, makes sense. Can you create and test it in your app and send me a pull request if everything checks out? I'd merge it back into master then

**Comment by erizzo** [comment]
I'm git-inept; if I just send the one .java file would that be acceptable?
